docs/install.md

```diff
@@ -74,3 +87,41 @@ The different supported modes can be set by using the `hook-mode` option:
 - `podinformer`: Use a Kubernetes controller to get information about new pods. This option is racy and the first events produced by a container could be lost. This mode is selected when `auto` is used and the above modes are not available.
 - `ldpreload`: Adds an entry in `/etc/ld.so.preload` to call a custom shared library that looks for `runc` calls and dynamically adds the needed OCI hooks to the cointainer `config.json` specification. This feature only works when runc is used. Since this feature is highly experimental, it'll not be considered when `auto` is used.
 - `nri`: Use the [Node Resource Interface](https://github.com/containerd/nri). It requires containerd v1.5 and it's not considered when `auto` is used.
+
+### Specific Information for Different Platforms
+
+This section explains the additional steps that are required to run Inspektor
+Gadget in some platforms.
+
+#### Minikube
+
+You should create the minikube cluster in different ways according to the gadget
+you want to use.
+
+##### Using a VM driver
+
+This option uses a VM driver (like Virtualbox or kvm2). It's compatible with all
+the gadgets that don't require BTF support. This option uses a custom minikube
+image that contains a more recent kernel version (5.4.40) and some features
+enabled to make eBPF programs work there. More details are available
+[here](https://github.com/kinvolk/cloud-native-bpf-workshop/blob/master/minikube.md#our-branch).
+
+The traceloop and network policy advisor gadgets only work with this driver.
```

Comment:
Does this mean that the other gadgets no loger work without BTF?

Reply:
No, it means that traceloop and network policy advisor can only be used with this driver, i.e. they don't work with the docker driver. Other gadgets that don't need BTF (all the traditional x-snoop ones from bcc) will work fine with this driver.

Reply:
Alright, in that case, I think this should have another sentence saying: "Non-BTF gadgets work as well", or maybe "The bcc-wrapped gadgets work as well". Or something, not sure how to group those in a useful manner.

Reply:
I changed the test to be simpler. I'm recommending the VM driver for traceloop and the network policy advisor, otherwise the docker driver is recommended.